scripts/gen_profiles_layer.py

```diff
@@ -4394,7 +4407,7 @@ def generate_string_to_format(self, formats):
 
     args = parser.parse_args()
 
-    registryPath  = './build/_deps/vulkan-headers-src/registry/vk.xml'
+    registryPath  = './external/Vulkan-Headers/build/install/share/vulkan/registry/vk.xml'
```

Comment:
Is that actually correct ? @juan-lunarg 
Actually, how could it even work without this... Oo

Reply:
I think this default path is only when working locally on the scripts so it's fine.

Reply:
Not a very clean approach TBH.

VVL passes the registry path to our scripts like so:
https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/f83f141f879d1d148a1c9b1026a9c724d4ad2bcf/layers/CMakeLists.txt#L157

See Vulkan-Headers doc:
https://github.com/KhronosGroup/Vulkan-Headers/blob/main/BUILD.md

Reply:
However, this approach is consistent with what was already being done. So it looks good to me.

Reply:
The `_deps` folder doesn't exist anymore so I just updated this